Cargo.toml

```diff
@@ -8,3 +8,7 @@ license = "Apache-2.0"
 repository = "https://github.com/sourcefrog/unix_mode"
 
 [dependencies]
+
+[target.'cfg(unix)'.dependencies]
```

Comment:
I think we can make this `dev-dependencies` as it's only needed for tests?

Reply:
done.

Reply:
Oh whoops! Thanks for catching that.